### ISSUE TITLE
Make app more accessible

### ISSUE THREAD

**Issue by bmlancien** [rl-institut/digiplan#66]
Right now the app has a 100% score on the Lighthouse accessibility tool in the browser, but there are still a few things that could/should be improved:

- Being able to navigate through all items with the tab key (right now the tabs map/charts are not accessible)
- Check if using the web accessibility features from echarts makes sense and if yes, implement them
- _Optional_: Think of a way to display map results/data to visually impaired users, e.g. display the results as a list that a user can go through with a screen reader (not sure if and how it is possible)

I also opened that issue to emphasise the importance of accessibility, especially because the RLI wants to be all inclusive. Web accessibility is quite a broad topic, so we won't be able to manage everything, but I think it is good to always have this on the back of our minds.
